ibm/service/catalogmanagement/data_source_ibm_cm_offering.go

```diff
@@ -562,6 +562,107 @@ func DataSourceIBMCmOffering() *schema.Resource {
 													Computed:    true,
 													Description: "Version source URL.",
 												},
+												"working_directory": &schema.Schema{
+													Type:        schema.TypeString,
```

Comment:
In datasource these attributes should be only computed.
We decalred in the schema but are we flattening and setting back?

Reply:
Changed them to computed only, thanks.

I'm not sure what you mean flattening and setting back? These have to be in the metadata because our API sets certain things in the metadata that aren't specified on a create or update.

Reply:
I changed all of the metadata to be computed, so I think that resolves your question.